src/main.rs

```diff
@@ -56,34 +59,63 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                println!("Fuzzing found errors!");
+                if let Some(ref mut terminal) = term_stdout {
+                    let _ = terminal.fg(term::color::YELLOW);
+                    println!("Fuzzing found errors!");
+                    let _ = terminal.reset();
+                } else {
+                    println!("Fuzzing found errors!");
+                }
                 process::exit(-1)
             }
         } else {
             result.map(|_| ())
         }
     } else if args.flag_list {
-        list_fuzz_targets()
+        list_fuzz_targets(&mut term_stdout)
             .map(|_| ())
     } else {
-        println!("Invalid arguments. Usage:\n{}", USAGE);
+        if let Some(ref mut terminal) = term_stdout {
+            let _ = terminal.fg(term::color::GREEN);
+            println!("Invalid arguments. Usage:\n{}", USAGE);
+            let _ = terminal.reset();
+        } else {
+            println!("Invalid arguments. Usage:\n{}", USAGE);
+        }
+
         return;
     };
     if let Err(error) = result {
-        writeln!(io::stderr(), "Error: {}", error)
-            .expect("failed writing to stderr");
+        if let Some(ref mut terminal) = term_stderr {
+            let _ = terminal.fg(term::color::RED);
```

Comment:
Same thing here, `Error:` should be red and bold, whereas the message should be white and bold.